README.md

```diff
@@ -1,73 +1,73 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Anime-World
 
-> Description the project.
+> "Anime-World" is a website show list of animes and show different facts about them.
 
+[Live Demo](scintillating-khapse-6ebe08.netlify.app)
```

Comment:
- [ ] Please do provide a valid deploy link to your project, the current link is broken🤔